### PR TITLE
fix(editor): UX Improvements to RBAC feature set

### PR DESCRIPTION
**Pull request by cstuncsik**
## Summary
Adding some UX tweaks RBAC and Projecs 

## Related tickets and issues
[PAY-1637](https://linear.app/n8n/issue/PAY-1637/ux-improvements-to-rbac-feature-set)

## Review / Merge checklist
- [x] PR title and summary are descriptive.
- [x] Tests included.

### CODE REVIEW

**Review by alexgrozav — DISMISSED** [review]

File: packages/editor-ui/src/components/CredentialCard.vue [hunk 1]
```diff
@@ -45,6 +45,9 @@ const uiStore = useUIStore();
 const credentialsStore = useCredentialsStore();
 const projectsStore = useProjectsStore();
 
+const resourceType = computed(
```
[the remark this answers is not in the document]
Comment (cstuncsik) [reply packages/editor-ui/src/components/CredentialCard.vue]: Yes, it's better to separate translation strings from keys
Done

File: packages/editor-ui/src/components/CredentialCard.vue [hunk 2]
```diff
@@ -45,6 +46,7 @@ const uiStore = useUIStore();
 const credentialsStore = useCredentialsStore();
 const projectsStore = useProjectsStore();
 
+const resourceTypeText = computed(() => locale.baseText('generic.credential').toLowerCase());
```
Comment (alexgrozav) [inline packages/editor-ui/src/components/CredentialCard.vue:49]: nit: `resourceTypeLabel` would've been more consistent as a name in all these places.

**Review by alexgrozav — APPROVED** [review]
LGTM ✅

**Review by alexgrozav — DISMISSED** [review]

File: packages/editor-ui/src/components/CredentialCard.vue (hunk 2 above)
Comment (cstuncsik) [reply packages/editor-ui/src/components/CredentialCard.vue]: Yes, makes sense, done

**Review by alexgrozav — DISMISSED** [review]

**Review by alexgrozav — APPROVED** [review]

**Pull request merged**